src/Service.Tests/SqlTests/RestApiTests/Find/FindApiTestBase.cs

```diff
@@ -823,7 +822,7 @@ public async Task FindTestWithPaginationVerifMultiplePrimaryKeysInAfter()
                 queryString: $"?$first=1",
                 entityNameOrPath: _entityWithCompositePrimaryKey,
                 sqlQuery: GetQuery(nameof(FindTestWithPaginationVerifMultiplePrimaryKeysInAfter)),
-                expectedAfterQueryString: $"&$after={Uri.EscapeDataString(SqlPaginationUtil.Base64Encode(after))}",
+                expectedAfterQueryString: $"&$after={SqlPaginationUtil.Base64Encode(after)}",
```

Comment:
I noticed some tests have `expectedAfterQueryString` base64 encode the after query param value directly, and some tests don't do that. Do you think it is worth making the tests consistent?

Reply:
if it doesn't take me too much time to get these consistent I will add it here, otherwise I will note it down to tackle later when there is more time.

Reply:
closing since this doesn't affect the test functionality and can come later.